src/lib/message/L1ToL2Message.ts

```diff
@@ -324,19 +320,23 @@ export class L1ToL2MessageReader extends L1ToL2Message {
 
         // We can skip by doing fromBlock.number + 1 on the first go
         // since creationBlock because it is covered by the `getAutoRedeem` shortcut
-        const redeemEventLogs = await this.l2Provider.getLogs({
```

Comment:
I think this has already been updated on c-nitro-stable, we need to do a merge here.